src/Microsoft.Health.Dicom.Core/Web/AcceptHeaderParameterNames.cs

```diff
@@ -10,4 +10,5 @@ public static class AcceptHeaderParameterNames
     public const string Type = "type";
     public const string TransferSyntax = "transfer-syntax";
     public const string Quality = "q";
+    public const string RequestOriginal = "msdicom-request-original";
```

Comment:
nit: x-ms-dicom-request-original?

Reply:
@wsugarman said x- is kind of deprecated usage. so removed it